doc/content/integrations/lora-cloud/das.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "LoRa Cloud Device & Application Services"
+description: ""
+---
+
+[LoRa Cloud™ Device & Application Services](https://www.loracloud.com/portal/device_management/home) comprise a set of full life-cycle management features for LoRaWAN®-based devices, which are intended to simplify the development of managed LoRaWAN endpoint solutions.
+
+<!--more-->
+
+LoRa Cloud Device & Application Services protocol lets you manage common device functionality at the application layer for all LoRa-based devices, and manage geolocation for modem-based devices (like [LoRa Edge™ LR1110 Solution](https://www.loracloud.com/documentation/device_management?url=overview.html#lora-edge-lr1110-solution)). These services also provide protocols for reliable application data reception from all Semtech's LoRaWAN-compatible devices, with efficient techniques to fight against packet loss. 
+
+{{< note >}} In general, LoRa Cloud Device & Application Services do not interact directly with network servers to retrieve uplinks or schedule downlinks, so it is usually client's responsibility to forward uplinks and schedule downlinks as a part of the response.
```

Comment:
I don't fully understand this. Who is the "client"?

Should we say that the user should configure the integration to enable LoRa Cloud Device & Application Services?